dbcon/mysql/ha_mcs_execplan.cpp

```diff
@@ -3725,9 +3725,7 @@ ArithmeticColumn* buildArithmeticColumn(
     const CalpontSystemCatalog::ColType& leftColType = pt->left()->data()->resultType();
     const CalpontSystemCatalog::ColType& rightColType = pt->right()->data()->resultType();
 
-    // Only tinker with the type if all columns involved are decimal
-    if (datatypes::isDecimalOperands(mysqlType.colDataType,
```

Comment:
datatypes::isDecimalOperands should be removed with this change. It was its only call.

Reply:
++